src/plugins/saved_objects_finder/public/finder/saved_object_finder.tsx

```diff
@@ -212,6 +213,12 @@ export class SavedObjectFinderUi extends React.Component<
 
   public render() {
     const { onChoose, savedObjectMetaData } = this.props;
+    const descriptionsDefined =
```

Comment:
I think we should make this check separately for each row item. So I would remove it here and change L317 (see comment)